demos/sample_azure_iot_pnp/sample_azure_iot_pnp.h

```diff
@@ -0,0 +1,107 @@
+/* Copyright (c) Microsoft Corporation.
+ * Licensed under the MIT License. */
+
+#ifndef SAMPLE_AZURE_IOT_PNP_H
+#define SAMPLE_AZURE_IOT_PNP_H
+
+#include <stdbool.h>
+#include <stdint.h>
+
+#include "azure_iot_hub_client_properties.h"
+
+// Header demo_config.h MUST define the Plug and Play Model ID

```

Comment:
>// [](http://example.com/codeflow?start=0&length=2)

can we use c based comment ?
/* */

Reply:
// is in the c99 spec. Why do you say it's not c based?

But if you want it to use a certain style, sure we can do it. Done.